presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcIntegrationSmokeTest.java

```diff
@@ -13,16 +13,130 @@
+            assertQueryFails(useUnknownType, format("INSERT INTO %s VALUES (1, null)", table.getName()), "Unsupported column type: unknown");
+            // TODO this query should pass, however it fails due https://github.com/prestosql/presto/issues/1185
+            assertQueryFails(
+                    useUnknownType,
+                    format("INSERT INTO %s (i) VALUES (1)", table.getName()),
+                    "UNKNOWN is not a valid type");
+
+            Session ignoreUnsupportedType = unsupportedTypeHandling(IGNORE);
+            assertQuery(ignoreUnsupportedType, "SELECT i FROM " + table.getName(), "VALUES 1");
+            assertQuery(ignoreUnsupportedType, "SELECT * FROM " + table.getName(), "VALUES 1");
+            assertQuery(
+                    ignoreUnsupportedType,
+                    "SELECT column_name, data_type FROM information_schema.columns WHERE table_name LIKE '%_unknown_%'",
+                    "VALUES ('i', 'integer')");
+            assertQuery(
+                    ignoreUnsupportedType,
+                    "DESCRIBE " + table.getName(),
+                    "VALUES ('i', 'integer', '', '')");
+
+            // TODO improve the error handling, current error is raised from
+            assertQueryFails(ignoreUnsupportedType, format("INSERT INTO %s (i) VALUES (2)", table.getName()), "(?s)Column count does not match.*");
+
+            Session convertToVarcharUnsupportedTypes = unsupportedTypeHandling(CONVERT_TO_VARCHAR);
+            assertQuery(convertToVarcharUnsupportedTypes, "SELECT i FROM " + table.getName(), "VALUES 1");
+            assertQuery(convertToVarcharUnsupportedTypes, "SELECT * FROM " + table.getName(), "VALUES (1, 'POINT (7 52)')");
+            assertQuery(
+                    convertToVarcharUnsupportedTypes,
+                    "SELECT column_name, data_type FROM information_schema.columns WHERE table_name LIKE '%_unknown_%'",
+                    "VALUES ('i', 'integer'), ('x', 'varchar')");
+            assertQuery(
+                    convertToVarcharUnsupportedTypes,
+                    "DESCRIBE " + table.getName(),
+                    "VALUES ('i', 'integer', '', ''), ('x', 'varchar', '','')");
+
+            // TODO improve the error handling, current error is raised from
+            assertUpdate(convertToVarcharUnsupportedTypes, format("INSERT INTO %s (i) VALUES (2)", table.getName()), 1);
+            assertUpdate(convertToVarcharUnsupportedTypes, format("INSERT INTO %s VALUES (3, null)", table.getName()), 1);
+            assertUpdate(convertToVarcharUnsupportedTypes, format("INSERT INTO %s VALUES (4, 'POINT (8 14)')", table.getName()), 1);
+            assertQueryFails(
+                    convertToVarcharUnsupportedTypes,
+                    format("INSERT INTO %s VALUES (5, 'wrong value')", table.getName()),
+                    "(?s)Data conversion error converting \"'wrong value'.*");
```

Comment:
Please add a test 
`SELECT * FROM .. WHERE x = 'wrong value'`

make sure PPD is disbaled then